### PR TITLE
Adds 2016 CR-V Executive (Euro version of Touring trim)

### PR DESCRIPTION
**Pull request by csouers**
Credit to @energee for the initial work. opendbc PR by @energee has already been merged in. Car is owned by discord user @noillek

Test route good to be made public. This is in tests already. "a859a044a447c2b0|2020-02-21--17-51-36"

### CODE REVIEW

**Comment by csouers** [comment]
Tests are currently failing for RAV4. Will retest with this in mind tonight.

**Comment by csouers** [comment]
@pd0wm this test is failing because the CRV for this PR is part of `candidates`. Not sure how you'd like to fix this.


```
FAIL: test_rav4_tss2 (tests.test_fw_fingerprint.TestFwFingerprint)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/openpilot/selfdrive/car/tests/test_fw_fingerprint.py", line 42, in test_rav4_tss2
    self.assertFingerprints(match_fw_to_car(CP.carFw), TOYOTA.RAV4_TSS2)
  File "/tmp/openpilot/selfdrive/car/tests/test_fw_fingerprint.py", line 14, in assertFingerprints
    self.assertEqual(len(candidates), 1)
AssertionError: 2 != 1

**Comment by csouers** [comment]
I believe this is ready to merge. Tests now pass with the new test route.

**Comment by csouers** [comment]
We discovered that steer threshold is different from most cars; much lower like on the Nidec RDX. Testing a new change over the next week. Will report back when the proper threshold is found.

**Comment by csouers** [comment]
Steer threshold of 400 is good according to the driver. Similar behavior to his prior Civics. Also fixed a rebase issue in values.py (line 138)

**Comment by csouers** [comment]
Latest cabana drive with the latest commits (different branch, same changes). https://my.comma.ai/cabana/?route=a859a044a447c2b0%7C2020-03-02--12-53-43&exp=1614717842&sig=YsgNLOsBllhlGD1ACiWVexKBP6H6c5iZGDVrx1ftZc0%3D&max=58&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2Fa859a044a447c2b0%2F27e03f44a369d41631b96c1ca0d9db2d_2020-03-02--12-53-43

File: selfdrive/test/test_car_models.py [hunk 1]
```diff
@@ -482,6 +487,7 @@ def get_route_log(route_name):
   "c9d60e5e02c04c5c|2018-01-08--16-01-49",  # HONDA CR-V 2016 TOURING
   "1632088eda5e6c4d|2018-06-07--08-03-18",  # HONDA CIVIC HATCHBACK 2017 SEDAN/COUPE 2019
   "fbd011384db5e669|2018-07-26--20-51-48",  # TOYOTA CAMRY HYBRID 2018
+  "a859a044a447c2b0|2020-02-21--17-51-36",  # HONDA CR-V 2016 EXECUTIVE
```
Comment (pd0wm) [inline selfdrive/test/test_car_models.py]: Can you remove it from this list, otherwise the test doesn't run

Comment (csouers) [reply selfdrive/test/test_car_models.py]: I have done this and a rebase. However, it looks like the test failed again. Maybe the route didn't make it to the bucket before it expired? I've changed it out for another.

Comment (csouers) [reply selfdrive/test/test_car_models.py]: I made the drive as of commit 4ebbd03 public in useradmin. It was created on the 3rd.

Comment (csouers) [reply selfdrive/test/test_car_models.py]: @pd0wm you see anything obvious why it isn't able to download the route for CRV_EU?

**Comment by csouers** [comment]
I changed the

**Comment by csouers** [comment]
@pd0wm you see anything obvious why it isn't able to download the route (in the commit) for CRV_EU? I made it public yesterday morning.

**Comment by csouers** [comment]
> @pd0wm you see anything obvious why it isn't able to download the route (in the commit) for CRV_EU? I made it public yesterday morning.

Looks like

**Comment by pd0wm** [comment]
Sorry for not responding earlier. I needed to sync the route into a special bucket. Rerunning the tests now.

**Comment by csouers** [comment]
> Sorry for not responding earlier. I needed to sync the route into a special bucket. Rerunning the tests now.

No worries. I have access to all of the user's drives, so if you need a fresh one, let me know.

**Pull request merged**